src/notes.ts

```diff
@@ -233,11 +244,9 @@ const openLink = (event: MouseEvent) => {
 
   event.preventDefault();
   const target = event.target as HTMLLinkElement;
-  if (!target || !target.href) {
-    return;
+  if (target && target.href && target.href.startsWith("http")) {
```

Comment:
Links starting with `chrome://` are ignored by default. We are explicitly adding a condition that only links with `http` can be open.